### DISCUSSION TITLE
Here is a compiled list of some feedback I have for the osu!editor

### DISCUSSION THREAD

**Discussion by OwenCMYK** [ppy/osu#26902] [Ideas]
I'm a new mapper who has been trying to get into mapping through the lazer osu!editor, I've noticed some flaws that make it borderline unusable though. As well I have some suggestions.

### None opinionated issues
- Putting sound effects on sliders won't play them in game, but they will still save and show up on the slider in the editor. This makes properly adding hitsounds to a map effectively impossible. (I think this is submitted as a bug already but I can't tell if they're referring to the same thing)
- There is no way to add breaks, or if there is, it's incredibly unclear.
- ~~Exporting to stable makes slider ends extend later, which ruins the rhythm obviously.~~ **Nevermind, I think this was a timing issue because I adjusted the at some point and I think I must have forgotten to re-snap certain objects to their beats.**

### Opinionated issues:
- There should be an option in the slider's inspector to have it reverse. I watched a YouTube video that happened to show the slider being dragged in the timeline, but otherwise I would have never known how to make reverse slider.
- The New Combo button looks like a circle button, and I think it should be changed to the number 1. I keep accidentally clicking it instead of switching to the circle hit object. I know this is definitely a me issue, but I'm sure others will go on to have the same issue, and the icon doesn't really represent a new combo well to begin with.
- The "Effect" button in the timing menu currently does not deserve to exist. All it does is bring the Kiai option out. Why not just have the Kiai option there to begin with? Also, does that really warrant its own box in that side panel?
- The way spinners are placed it really unintuitive and seems like a bug until you figure out how it works. Once you start placing it, it's unclear that the placing process isn't finished. Maybe have objects that are in the process of being placed appear with a flashing yellow outline in the timeline, that way after they've placed a slider, they will be able to intuit the placement behavior when seeing the same outline on the spinner.

I hope at least some of this is helpful. Let me know if there's a better way I should suggest these.

**Comment by bdach** [comment]
> Exporting to stable makes slider ends extend later, which ruins the rhythm obviously.

Can you elaborate on this? I assume you're exporting "for compatibility"?

**Reply by OwenCMYK** [reply]
Yes, I exported "for compatibility" and when I imported it into stable, the slider ends were slightly later than they were supposed to be. If you go back into the editor and move the points slightly then they'll resnap their timing. I can probably send a beatmap file as an example if you'd like.

**Reply by OwenCMYK** [reply]
Actually, I stand corrected. I think this may have been an issue on my part. I apologize for the potentially false information.
